semantic-release/action.yaml

```diff
@@ -43,6 +43,12 @@ outputs:
   new-release-patch-version:
     description: "Patch version of the new release"
     value: ${{ steps.semantic-release.outputs.new-release-patch-version }}
+  last-release-version:
```

Comment:
Sort these alphabetically

Reply:
Update the readme adding these to it (we haven't automated this - not sure if there is a way)